pkg/scalers/apache_kafka_scaler.go

```diff
@@ -0,0 +1,682 @@
+package scalers
```

Comment:
Could you please add header here? check other scalers.

Also could you please put a comment below the header explaining this scaler (why we have 2 kafka scalers).

Thanks!

Reply:
This does seem to be lacking in a lot of scalers.